packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -1026,5 +1026,12 @@ module.exports.external = [
       }
       return JSON.stringify(referenceData).slice(1, -1);
     }
-  }
+  },
+  {
+    name: "personalRelationshipRoleTypeCodeSystem",
+    description: "Returns the code system for the related person relationship code",
+    func: function () {
+      return "2.16.840.1.113883.1.11.19563";
+    },
+  },
 ];
```

Comment:
this isn't being used anywhere?

Reply:
In relatedPersonCodeFromText. It's a large diff so doesn't show up automatically

Reply:
nit: could define this in a const and put it at the top of the file and use it here

Reply:
handlebars doesnt have constants, but I modified it to use a single with at the top of the file and then passed that as context to each eq comparison

Reply:
I don't see what you're referring to

I just mean to add `const personalRelationshipTypeCode = "2.16.840.1.113883.1.11.19563";` to the top of `handlebars-helpers.js`, and then use that const in this func

Reply:
ah so I was referring to the use of personalRelationshipRoleTypeCodeSystem within `RelatedPersonCodeFromText`. 
Made the change you were referring to as well